jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/ResponseHttpFields.java

```diff
@@ -203,4 +242,59 @@ public String toString()
     {
         return _fields.toString();
     }
+
+    /**
+     * A marker interface for {@link HttpField}s that cannot be {@link #remove(HttpHeader) removed} or {@link #clear() cleared}
+     * from a {@link ResponseHttpFields} instance.   Persistent fields are not immutable in the {@link ResponseHttpFields}
+     * and may be replaced with a different value.
+     */
+    public interface PersistentField
```

Comment:
I'd rename it to just `Persistent` as it is already part of the outer class so `ResponseHttpFields.Persistent`, and does not duplicate `Field`.